deps/wazuh_testing/wazuh_testing/tools/__init__.py

```diff
@@ -91,28 +91,34 @@
 
 
 def get_version():
+    try:
+        if platform.system() in ['Windows', 'win32']:
+            with open(os.path.join(WAZUH_PATH, 'VERSION'), 'r') as f:
+                version = f.read()
+                return version[:version.rfind('\n')]
 
-    if platform.system() in ['Windows', 'win32']:
-        with open(os.path.join(WAZUH_PATH, 'VERSION'), 'r') as f:
-            version = f.read()
-            return version[:version.rfind('\n')]
-
-    else:  # Linux, sunos5, darwin, aix...
-        return subprocess.check_output([
-          f"{WAZUH_PATH}/bin/wazuh-control", "info", "-v"
-        ], stderr=subprocess.PIPE).decode('utf-8').rstrip()
+        else:  # Linux, sunos5, darwin, aix...
+            return subprocess.check_output([f"{WAZUH_PATH}/bin/wazuh-control", "info", "-v"],
+                                           stderr=subprocess.PIPE).decode('utf-8').rstrip()
+    except Exception:
+        return 'N/A'
```

Comment:
Specify an exception

Reply:
Why are you returning 'N/A' and not False or something more descriptive? Could we choose a better way here?

Reply:
Due to the function is called get_version it maybe has not sense to return false, but I agree this is could lead into errors in the future. However I can not see a perfect solution here, due to legacy code.
Maybe something like the following could make the job:
```
def get_version():
    try:
        if platform.system() in ['Windows', 'win32']:
            with open(os.path.join(WAZUH_PATH, 'VERSION'), 'r') as f:
                version = f.read()
                return version[:version.rfind('\n')]

        else:  # Linux, sunos5, darwin, aix...
            return subprocess.check_output([f"{WAZUH_PATH}/bin/wazuh-control", "info", "-v"],
                                           stderr=subprocess.PIPE).decode('utf-8').rstrip()
    except FileNotFoundError:
        return 'Version file not found'
    except subprocess.CalledProcessError as e:
        return f'Error getting version: {e}'
    except PermissionError as e:
        return f'Permission error: {e}'
```

It is not the perfect solution, but we need to make this function not fail in case of failure gathering the version